hoot-test/src/main/cpp/hoot/test/main.cpp

```diff
@@ -83,18 +85,19 @@ using TestPtr = std::shared_ptr<CppUnit::Test>;
 
 enum _TestType
 {
-  NONE          = 0x00,
-  CURRENT       = 0x01,
-  QUICK_ONLY    = 0x02,
-  QUICK         = CURRENT | QUICK_ONLY,
-  SERIAL        = 0x04,
-  CASE_ONLY     = 0x08,
-  SLOW_TESTS    = 0x10,
-  SLOW_ONLY     = SLOW_TESTS | SERIAL | CASE_ONLY,
-  SLOW          = SLOW_ONLY | QUICK,
-  GLACIAL_TESTS = 0x20,
-  GLACIAL_ONLY  = GLACIAL_TESTS | SERIAL,
-  GLACIAL       = GLACIAL_ONLY | SLOW
+  NONE           = 0x00,
+  CURRENT        = 0x01,
+  QUICK_ONLY     = 0x02,
+  QUICK          = CURRENT | QUICK_ONLY,
+  SERIAL         = 0x04,
+  CASE_ONLY      = 0x08, // conflate case tests
+  VALIDATED_ONLY = 0x10,
```

Comment:
Added an option to just run tests that do any output validation. Useful when debugging validation problems in the tests or updating validation report files.